grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -491,7 +503,13 @@ export class ClientPageFacade {
     this.store.stopFlowConfiguration();
   }
 
+  /** Adds a label to the selected client */
   addClientLabel(label: string) {
     this.store.addClientLabel(label);
   }
+
+  /** Removes a label from the selected client */
+  removeClientLabelReq(label: string): Observable<HttpResponse<{}>> {
+    return this.store.removeClientLabel(label);
+  }
```

Comment:
I find it a bit awkward that we have a `addClientLabel` method (that returns nothing) and a `removeClientLabelReq` method (that returns an observable). Is this inconsistency necessary?

Reply:
Totally agree here, but in order to show the `UNDO` snackbar we needed to know if the remove request is successful or not. If there is a better way to achieve this behaviour, I am open to suggestions.

Reply:
@max-vogler should have some suggestions.